pkg/amazon/backend/convert.go

```diff
@@ -3,11 +3,15 @@ package backend
 import (
 	"encoding/json"
 	"fmt"
+	"os"
+	"path/filepath"
 	"sort"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/joho/godotenv"
```

Comment:
Need to sort imports. If you use gopls or goimports this should work automagically.

Reply:
I havent found how to configure Golang so it doesn't split imports like this